releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1500,7 +1500,7 @@
   "124055": {
     "commit": "846e282d0523fdb23522e6b318186aa80593c594",
     "text": "Optimized log output to avoid printing out redundant information of the pod.",
-    "markdown": "Optimized log output to avoid printing out redundant information of the pod. ([#124055](https://github.com/kubernetes/kubernetes/pull/124055), [@yangjunmyfm192085](https://github.com/yangjunmyfm192085)) [SIG Scheduling]",
+    "markdown": "Optimized log output to avoid printing out redundant information of the pod. ([#124055](https://github.com/kubernetes/kubernetes/pull/124055), [@yangjunmyfm192085](https://github.com/yangjunmyfm192085))",
```

Comment:
```suggestion
    "markdown": "Optimized log output to avoid printing out redundant information of the pod. ([#124055](https://github.com/kubernetes/kubernetes/pull/124055), [@yangjunmyfm192085](https://github.com/yangjunmyfm192085)) [SIG Scheduling]",
```